src/proto/macro.proto

```diff
@@ -0,0 +1,13 @@
+syntax = "proto2";
+
+package mixx.track.io;
+
+option optimize_for = LITE_RUNTIME;
+
+message Macro {
+  message Action {
+    optional uint64 origin = 1;
+    required bytes data = 2;
+  }
```

Comment:
We definitely need a type, so that we open up the possibility to add actions other then hotcue jumps later on.

Another thing: AFAIK required fields are considered an anti-pattern in protobuf v2 and removed in v3, so we should avoid them.

Reply:
You're right in both regards. The data is also a generic "bytes" field for now to keep it open. We'll see.